src/main/java/slt/rest/ImportService.java

```diff
@@ -87,7 +87,8 @@ public ResponseEntity setAll(@RequestBody Export export) {
                 });
 
         for (SettingDto settingDto : settingDtos) {
-            settingsRepo.putSetting(userInfo.getUserId(), settingDto.getName(), settingDto.getValue(), settingDto.getDay());
+            Setting setting = myModelMapper.getConfiguredMapper().map(settingDto, Setting.class);
```

Comment:
nu wordt ID ook meegemapped. Ik dacht dat we de id's niet in de import/export wilden.

Sidenote: import export is nu ondergeschikt, dus misschien niet echt van belang atm

Reply:
Nee goed punt, kan handmatig wel op null gezet worden zodat we deze functie toch kunnen gebruiken. Mappers ftw